windows/src/desktop/help/basic/config_tasks/config_menu.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Keyman Configuration {#basic_config_menu}
+---
+
+Keyman Configuration is the central control panel for Keyman. Use Keyman
+Configuration to purchase and activate Keyman, switch the language of
+Keyman menus, install and uninstall keyboards, change options, set
+hotkeys, disable addins, associate languages, check for updates and run
+support diagnostics.
+
+![](desktop_images/tab-support.png)
```

Comment:
Are the desktop_images up two folders? also for l. 17
`../../desktop_images/...png`